utils/models_to_deprecate.py

```diff
@@ -124,19 +128,21 @@ def get_list_of_models_to_deprecate(
         }
 
         # We make successive calls to the hub, filtering based on the model tags
-        n_seen = 0
-        for model, model_info in models_info.items():
+        print("Making calls to the hub to find models below the threshold number of downloads")
+        num_models = len(models_info)
+        for i, (model, model_info) in enumerate(models_info.items()):
+            if max_num_models != -1 and i > max_num_models:
+                break
+            print(f"{i+1}/{num_models}: getting hub downloads for {model_info['tags']}")
             for model_tag in model_info["tags"]:
                 model_list = HubModelLister(tags=model_tag)
-                for i, hub_model in enumerate(model_list):
-                    n_seen += 1
-                    if i % 100 == 0:
-                        print(f"Processing model {i} for tag {model_tag}")
-                    if max_num_models != -1 and i > n_seen:
-                        break
+                for hub_model in model_list:
```

Comment:
Not a big deal, but do you think it's ever possible a single repository could have more than 1 (model arch.) tags?

Reply:
and regarding

```
            for model_tag in model_info["tags"]:
                model_list = HubModelLister(tags=model_tag)
```

I think `api.list_models` could take a list of tags

```
    def list_models(
        self,
        *,
        # Search-query parameter
        filter: Union[str, Iterable[str], None] = None,
```
so maybe we just do

```
            model_list = HubModelLister(tags=model_info["tags"])
```
without the for loop?